dll/win32/kernel32/kernel32.spec

```diff
@@ -483,7 +483,10 @@
 @ stdcall -version=0x600+ GetFinalPathNameByHandleA(ptr str long long)
 @ stdcall -version=0x600+ GetFinalPathNameByHandleW(ptr wstr long long)
 @ stdcall GetFirmwareEnvironmentVariableA(str str ptr long)
+@ stdcall -version=0x602+ GetFirmwareEnvironmentVariableExA(str str ptr long long)
+@ stdcall -version=0x602+ GetFirmwareEnvironmentVariableExW(wstr wstr ptr long long)
 @ stdcall GetFirmwareEnvironmentVariableW(wstr wstr ptr long)
+@ stdcall -version=0x602+ GetFirmwareType(ptr)
```

Comment:
Afaik UEFI support was introduced with WinVista NT6.0 already.
So are you sure that those funcs shouldn't be -version=0x600+ instead? Same question for the ifdefs in winbase.h?

I can see that MSDN claims they are Win8+,
https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getfirmwaretype
but I don't trust the MSDN regarding MinSupportedClient. Therefore I do explicitly raised this question here.

Reply:
Yes, that's right.

NT5.2 introduced:
(Get/Set)FirmwareEnvironmentVariable(A/W)

NT6.2 introduced:
(Get/Set)FirmwareEnvironmentVariableEx(A/W)
GetFirmwareType

NT5.2 is already implemented UEFI firmware variable APIs, and NT6 supported UEFI in low-level, then NT6.2+ introduced more APIs and features (like shutdown /fw, advance boot, ...) about UEFI.

Reply:
@JoachimHenze EFI support was introduced in Vista / windows 2003 Itanium 

UEFI was introduced with windows 8 
 
 they're different specs, and need massive changes to support either or.

Reply:
Okay, thx for clarification. Feel free to move ahead in the way you believe it makes most sense. I just wanted to prevent any oopsies. But it seems to me you know what you are doing.